src/instances.ts

```diff
@@ -1,4 +1,4 @@
-import chalk, { Chalk } from 'chalk';
+import chalk = require('chalk');
```

Comment:
Could we avoid using `require` imports please?

Reply:
done.